docker/docker-compose.yml

```diff
@@ -80,15 +80,12 @@ services:
     volumes:
       - /dev/shm:/dev/shm
 
-  # @todo #142:60m Resurrect coala.
-  #  Already done it at se#302.
   lint:
     image: fidals/coala-ci
-    env_file:
-      - env_files/paths
     working_dir: $SRC_DIR
-#    volumes:
-#      - ./../:$SRC_DIR
+    command: coala
+    volumes_from:
+      - app
```

Comment:
@duker33 
>By the way, this not little PR was merged without review approval without explaining why.

Sorry, my bad. As it seemed to me, the changes were not so huge and important, because we already implemented the same in SE, so I decided to merge them

Please feel free to comment this PR if you will find something else. I am going to create a PR with fixes.

Reply:
@artemiy312 , my (our, i think :)) practice shows, that copypasting is not less dangerous, then development.
Let's do reviews, if it's not about only two strings and much hurry